lib/fencing.py.py

```diff
@@ -879,6 +879,27 @@ def set_multi_power_fn(connection, options, set_power_fn, get_power_fn, sync_set
 
 	return False
 
+def multi_reboot_cycle_fn(connection, options, reboot_cycle_fn, retry_attempts=1):
+	success = True
+	plugs = options["--plugs"] if "--plugs" in options else [""]
```

Comment:
It looks like this still passes in the whole plug argument instead of one at a time. Shouldn't this be:
plugs = options["--plugs"].split(",") if "--plugs" in options else [""]